CHANGELOG.md

```diff
@@ -10,6 +10,9 @@ Current
 
 ### Added:
 
+- [Add custom dimension row serializer](https://github.com/yahoo/fili/pull/399)
+    * Add custom dimension row serializer so that each dimension row serializes into dimension row cache correctly
+
```

Comment:
I suggest you move this to a fix and describe that this resolves where dimension field names do not serialize correctly.